components/SidebarItem.tsx

```diff
@@ -19,13 +19,15 @@ const SidebarItem = ({ route, index, asPath, activeItem }) => {
                         ? '2px solid var(--primary_light)'
                         : '2px solid var(--docs_border_strong)',
                     display: 'flex',
-                    alignItems: 'center',
+                    alignItems: 'flex-start',
                     paddingLeft: '1rem',
                     marginLeft: '0.95rem'
                 }}>
                 <MinusIcon
                     style={{
                         width: '12px',

```

Comment:
icon has size prop as well as height and width prop can use that as it will decrease and increase svg too